### ISSUE TITLE
Support Azure AD Workload Identity

### ISSUE THREAD

**Issue by stevehipwell** [thanos-io/objstore#54]

**Comment by fpetkovski** [comment]
Good idea, moved.

**Comment by stevehipwell** [comment]
@phillebaba @fpetkovski don't we need an issue in both repos? This repo needs an issue to track the actual work while the main Thanos repo needs an issue to track when Azure AD Workload Identity support has been added into Thanos.

**Comment by cmergenthaler** [comment]
Any update on this?

**Comment by rikhil-s** [comment]
@phillebaba Any further updates on this? AAD Pod Identity is now end of life so we now have to use Thanos with the Azure Workload Identity Sidecar (which also won't be supported long-term).

**Comment by rikhil-s** [comment]
PR has been created here #82 to resolve this issue. Just need someone to review!

**Comment by MichaHoffmann** [comment]
Yeah i think this issue can be closed! It can be tested with 0.33.0-rc.0 already
